kubernetes/product-deployment/scripts/testgrid-rds-cf.yaml

```diff
@@ -0,0 +1,48 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Parameters:
+  pDbUser:
+    Type: String
+  pDbPass:
+    Type: String
+  pDbEngine:
+    Type: String
+  pDbVersion:
+    Type: String
+  pDbInstanceClass:
+    Type: String
+
+Resources:
+  TESTGRIDDB:
+    Type: AWS::RDS::DBInstance
+    Properties:
+      MasterUsername: !Ref pDbUser
+      MasterUserPassword: !Ref pDbPass
+      Engine: !Ref pDbEngine
+      EngineVersion: !Ref pDbVersion
+      DBInstanceClass: !Ref pDbInstanceClass
+      StorageType: gp2
+      PubliclyAccessible: True
+      AllocatedStorage: "20"
+      DBInstanceIdentifier: testgrid-rds
+      LicenseModel:
+        !If [UseLicensedVersion, license-included, !Ref "AWS::NoValue"]
+      Tags:
+        - Key: email
+          Value: tharsanan@wso2.com
```

Comment:
can we remove these tags as it contains your email? Maybe a different tag will help.

Reply:
just a suggestion, without just putting the product name we can append a keyword like `Kubenetes deployment`.

Reply:
Updated code and passing the tag name from this [line](https://github.com/wso2/testgrid-jenkins-library/blob/47fc6e51668474881fff4152fd7fb871d1551564/kubernetes/product-deployment/scripts/create-database.sh#L52)